kotlinx-coroutines-core/native/src/internal/Concurrent.kt

```diff
@@ -32,6 +32,6 @@ internal open class SuppressSupportingThrowableImpl : Throwable() {
     }
 }
 
-@SharedImmutable
 @OptIn(ExperimentalStdlibApi::class)
```

Comment:
```suggestion
// getter instead of a property due to the bug in the initialization dependencies tracking with '-Xir-property-lazy-initialization=disabled' that Ktor uses 
@OptIn(ExperimentalStdlibApi::class)
```